src/ui/CustomStyleEditor.tsx

```diff
@@ -274,7 +275,7 @@ export class CustomStyleEditor extends React.PureComponent<any, any> {
               this.state.styles.prefixValue
             )}</strong>${textSample}`;
           } else {
-            sampleDiv.innerText = `${this.getNumberingLevel(
+            sampleDiv.innerHTML = `${this.getNumberingLevel(
```

Comment:
Why this change? The value assigned doesn't look like html, and if it does need to be html, it should use dom elements instead of innerHTML to avoid injection attacks.

Reply:
This update addresses ticket KNITE-1292: Fix for <strong> tag issue in custom styles.

Previously, when the bold style was applied to the first word or sentence, the <strong> tag would be added incorrectly at the word or sentence level instead of applying the bold formatting. This issue has now been resolved with this fix.

Reply:
@abin-aloysious as far as I can tell though innerText will suffice here. If you must add html, use document.createElement + append. You can only use innerHTML with static hardcoded strings for security reasons.